src/clients/SkillServiceClient.ts

```diff
@@ -2,26 +2,48 @@ import {PowerHttpClient} from './PowerHttpClient';
 import {APISkillServiceSkill} from '../model/skill/SkillServiceSkill';
 
 import axios, {AxiosRequestConfig} from 'axios';
-import {getSkillByName} from '../API_CONFIG';
 import {APISkillCategory} from '../model/skill/SkillCategory';
 
 declare const POWER_SKILL_SERVICE_URL: string;
 
 export class SkillServiceClient extends PowerHttpClient {
 
+    private static _instance: SkillServiceClient;
+
+    private constructor() {
+        super();
+    }
+
+    public static instance(): SkillServiceClient {
+        if (!this._instance) {
+            this._instance = new SkillServiceClient();
+        }
+        return this._instance;
+    }
+
     private base = (): string => {
         return POWER_SKILL_SERVICE_URL;
     };
 
     public getSkillByName = (name: string): Promise<APISkillServiceSkill> => {
-        const url = this.base() + "/skill/byName";
+        const url = this.base() + '/skill/byName';
         const config: AxiosRequestConfig = {params: {qualifier: name}};
-        return this.executeRequest(axios.get(getSkillByName(), config));
+        return this.executeRequest(axios.get(url, config));
     };
 
     public categorizeSkill = (name: string): Promise<APISkillCategory> => {
-        const url = this.base() + "/skill";
+        const url = this.base() + '/skill';
         const config: AxiosRequestConfig = {params: {qualifier: name}};
         return this.executeRequest(axios.post(url, null, config));
     };
+
+    public addSkillVersion = (skillId: number, version: string): Promise<string[]> => {
```

Comment:
Ich komm grad nicht aufs Dev-System, aber die Verwendung der Version in der URL kann Probleme geben. Hast du mal angeschaut, wie sich eine Version mit z.B. einem "?" oder "/" Verhält? Man kann sich damit auch lustige Encoding-Fehler Einhandeln 
Pack das lieber in den Request Body, es ist ja schon ein Post.
axios.post(url, {version})